generate-dockerignore.gha.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I am not sure if this script is required?

Reply:
This is a draft PR, so this isn't final. Either way, the reason for this script was so that we could have separate `.dockerignore` files for local dev and github actions. This is required because the local `.dockerignore` files attempt to ignore `.jest.config.js` files, which stops the api-test QAT pipelines from running